jgrapht-core/src/main/java/org/jgrapht/alg/util/Extension.java

```diff
@@ -20,7 +20,7 @@
  * the Eclipse Foundation.
  */
 /* -----------------
- * MaximumFlowAlgorithmBase.java
```

Comment:
I'm confused...shouldn't this file Extension.java be deleted?  Why is it still present in the pull request?